### ISSUE TITLE
version 3.2 no longer working?

### ISSUE THREAD

**Issue by neobie** [Stichoza/google-translate-php#115]
Just curious. version 3.2 no longer working? resulting in 400 error ?

**Comment by frzsombor** [comment]
Google changed a few things around 23rd July 2019, and it has been fixed in the latest v4.x version, but not in the old v3.2 branch. There is a pending [pull request (#114)](https://github.com/Stichoza/google-translate-php/pull/114) right now, based on the fixes of the current version, but it hasn't been accepted yet by the project owner. If you need a quick solution, start using the latest version, or fix the code manually, based on the [changes](https://github.com/Stichoza/google-translate-php/pull/114/files) in the pull request referenced above.

**Comment by gbelot2003** [comment]
I have the same problem, imposible to upgrate to +7.1

**Comment by Qh0stM4N** [comment]
@neobie No working but you cant auto upragate with composer.
https://github.com/Stichoza/google-translate-php/pull/114#issuecomment-557043845
if you don't have enough time for v3.2.15 version? you can hack composer.lock files manually
``` json
{
    "name": "stichoza/google-translate-php",
    "version": "v3.2.14",
    "source": {
        "type": "git",
        "url": "https://github.com/Stichoza/google-translate-php.git",
        "reference": "a12ff845b1ec3870795a6fb1cf841f7a1585feba"
    },
    "dist": {
        "type": "zip",
        "url": "https://api.github.com/repos/Stichoza/google-translate-php/zipball/a12ff845b1ec3870795a6fb1cf841f7a1585feba",
        "reference": "a12ff845b1ec3870795a6fb1cf841f7a1585feba",
        "shasum": ""
    },
    "require": {
        "guzzlehttp/guzzle": "~6.0",
        "php": ">=5.5.0"
    },
    "require-dev": {
        "phpunit/phpunit": "~4.0"
    },
    "type": "library",
    "autoload": {
        "psr-0": {
            "Stichoza\\GoogleTranslate\\": "src/"
        },
        "psr-4": {
            "Stichoza\\GoogleTranslate\\": "src/Stichoza/GoogleTranslate/"
        }
    },
    "notification-url": "https://packagist.org/downloads/",
    "license": [
        "MIT"
    ],
    "authors": [
        {
            "name": "Levan Velijanashvili",
            "email": "me@stichoza.com"
        }
    ],
    "description": "Free Google Translate API PHP Package",
    "homepage": "http://github.com/Stichoza/google-translate-php",
    "keywords": [
        "google",
        "php",
        "translate"
    ],
    "time": "2018-06-26T08:24:07+00:00"
},
```
Maybe  your old version hash code different this.
![image](https://user-images.githubusercontent.com/12828465/73726412-8ef32580-4740-11ea-811d-f2275ebcd518.png)

**Comment by Stichoza** [comment]
Just released v3.2.15 for the 3.2 branch, but please use the latest version of this package
